server/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -1367,7 +1381,8 @@ public void testTranslogWriterCanFlushInAddOrReadCall() throws IOException {
             temp.getTranslogPath(),
             temp.getIndexSettings(),
             temp.getBigArrays(),
-            new ByteSizeValue(1, ByteSizeUnit.KB)
+            new ByteSizeValue(1, ByteSizeUnit.KB),
+            RANDOMIZING_IO_BUFFERS
```

Comment:
Can we randomly use `DiskIoBufferPool.INSTANCE` too?

Reply:
++ done